maintenance/projects/js-toolkit/packages/liferay-npm-build-tools-common/src/project/index.ts

```diff
@@ -362,18 +375,18 @@ export class Project {
 		}
 	}
 
-	_findAutopresets(): string[] {
+	_getAutopreset(): string | null {
 		const {dependencies = {}, devDependencies = {}} = this._pkgJson;
 
-		return Object.keys({
+		const autopresets = Object.keys({
 			...dependencies,
 			...devDependencies,
```

Comment:
Are these the autopresets?
```
Object.keys({
	...dependencies,
	...devDependencies,
)};
```
Or are the reduced object keys the autopresets, just nitpicking here to perhaps reduce chaining idk 🤷

Reply:
They are the package names of all dependencies that are autopresets (i.e: have `liferay-npm-bundler` as a dependency).

:thinking: Maybe we should add another criterion to detect an autopreset, namely that it is a valid preset (its main index is a JSON file and it looks like a preset).

I will add it, I think :+1: